### PR TITLE
Documentation for installation on Linux

### LINKED ISSUE #806: Linux Upgrades
On Linux we currently have deb/rpm as install options while we are also working on AppImage. This feature is to provide an upgrade mechanism for each of the Linux install options we provide. For example, for deb/rpm this would likely be appropriate repos.

### PR DESCRIPTION
**Pull request by adamkpickering**
Note that while `stable` is the channel in the documentation, it has not been tested because there are no packages there at the moment. After we release 0.7.0 there should be packages there - not sure if we want to wait until that's the case to call this done. This also means that **at the moment, testing the commands as they are in this PR will not work**. If you want to test the commands, please substitute `dev` for every instance of `stable` in the commands. This will cause you to use the `dev` channel, which does have packages available.

### CODE REVIEW

**Comment by rak-phillip** [comment]
I vote that we wait until packages are published before we merge this one just to limit the possibility of confusion. 

With that said, I was able to test `.deb` and `.rpm` using the dev channel in Ubuntu 20.04 & openSUSE Tumbleweed and the instructions got me through with ease. Thanks for the change!

File: docs/installing.md
Comment (rak-phillip) [inline docs/installing.md:140]: I personally prefer the Oxford comma `, and`

File: docs/installing.md
Comment (rak-phillip) [inline docs/installing.md:162]: Oxford comma `, and`

File: docs/installing.md
Comment (rak-phillip) [inline docs/installing.md:166]: > To add `the` repository...

File: docs/installing.md
Comment (rak-phillip) [inline docs/installing.md:188]: Missing a comma after desktop

**Review by rak-phillip — COMMENTED** [review]
Instructions were straightforward and got me set up with a working Rancher Desktop install. There's just a few grammatical changes I would recommend.

**Review by gunamata — COMMENTED** [review]
I tried the .deb instructions on ubuntu but couldn't get it to work. It's not a issue with the steps provided, it's some issue connecting to or pulling from repository. I think it would help some users if we provide instructions to manually download the .deb file from github releases page and then run `sudo dpkg` etc?

File: docs/installing.md
Comment (mattfarina) [reply docs/installing.md:140]: I prefer the oxford comma, too.

**Comment by mattfarina** [comment]
@gunamata the directions won't work until we have a stable release with it. You can change the URL to the dev location (https://download.opensuse.org/repositories/isv:/Rancher:/dev/) or replace stable with dev and you should be able to test.

**Review by evertonlperes — DISMISSED** [review]
LGTM - Nicely done.
Tested under `dev` instead `stable`

File: docs/installing.md
Comment (ericpromislow) [inline docs/installing.md:132]: On two separate systems, an old one and a new 20.04.3 LTS, when I try to run this command, I get the error message `gpg: no valid OpenPGP data found.`  Looking at the output of the curl command is a 404. Highlights:
```
<title>Resource is no longer available!</title>
...
<body>
<h1>Resource is no longer available!</h1>
<p>


    The requested URL is no longer available on this server and there is no
    forwarding address.



    If you followed a link from a foreign page, please contact the
    author of this page.



</p>
<p>
If you think this is a server error, please contact
the <a href="mailto:webmaster@opensuse.org">webmaster</a>.

</p>

<h2>Error 404</h2>
```

**Review by ericpromislow — CHANGES_REQUESTED** [review]

File: docs/installing.md
Comment (ericpromislow) [inline docs/installing.md:133]: Ran the following commands:
```
curl https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb/Release.key | sudo apt-key add -
sudo add-apt-repository 'deb https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb/ ./'
```
and the last command gave the error message:

```
E: The repository 'https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb ./ Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
There was a space between `/stable/dev/` and `./'`, like in the instructions

If I try to continue anyway (which I did), running `sudo apt install rancher-desktop` gave the error message `E: Unable to locate package rancher-desktop`

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Couldn't get the `sudo add-apt-repository` command to work on ubuntu

File: docs/installing.md
Comment (evertonlperes) [reply docs/installing.md:133]: Looks like the commands are working fine on my Ubuntu VM. Performed uninstall/install twice and it seems to be working properly. Just need to replacing `stable` to `dev` in all commands:

```bash
curl https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb/Release.key | sudo apt-key add -

sudo add-apt-repository 'deb https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb/ ./'

sudo apt update

sudo apt install rancher-desktop
```


Full install log:
```bash
curl https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb/Release.key | sudo apt-key add -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1106  100  1106    0     0   1353      0 --:--:-- --:--:-- --:--:--  1352
OK
❯ sudo add-apt-repository 'deb https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb/ ./'
Hit:1 https://deb.nodesource.com/node_14.x focal InRelease
Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]                                                          
Hit:3 http://ca.archive.ubuntu.com/ubuntu focal InRelease                                                      
Get:4 http://ca.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]                                     
Get:5 https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb ./ InRelease [1,126 B]                              
Get:6 http://ca.archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]                                               
Get:7 https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb ./ Packages [473 B]
Get:8 http://ca.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [1,393 kB]
Get:9 http://security.ubuntu.com/ubuntu focal-security/main i386 Packages [344 kB]
Get:10 http://ca.archive.ubuntu.com/ubuntu focal-updates/main i386 Packages [572 kB]
Get:11 http://ca.archive.ubuntu.com/ubuntu focal-updates/main Translation-en [282 kB]    
Get:12 http://ca.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [278 kB]
Get:13 http://ca.archive.ubuntu.com/ubuntu focal-updates/main amd64 c-n-f Metadata [14.6 kB] 
Get:14 http://ca.archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [608 kB] 
Get:15 http://ca.archive.ubuntu.com/ubuntu focal-updates/restricted Translation-en [87.0 kB]
Get:16 http://ca.archive.ubuntu.com/ubuntu focal-updates/universe i386 Packages [652 kB]    
Get:17 http://ca.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [881 kB] 
Get:18 http://ca.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [362 kB]
Get:19 http://ca.archive.ubuntu.com/ubuntu focal-updates/universe amd64 c-n-f Metadata [19.6 kB]
Get:20 http://ca.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 DEP-11 Metadata [940 B]
Get:21 http://ca.archive.ubuntu.com/ubuntu focal-backports/main amd64 DEP-11 Metadata [7,976 B]
Get:22 http://ca.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [11.3 kB]
Get:23 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [1,064 kB]
Get:24 http://security.ubuntu.com/ubuntu focal-security/main Translation-en [196 kB]
Get:25 http://security.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [35.6 kB]
Get:26 http://security.ubuntu.com/ubuntu focal-security/main amd64 c-n-f Metadata [9,076 B]
Get:27 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [665 kB]
Get:28 http://security.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [64.4 kB]
Get:29 http://security.ubuntu.com/ubuntu focal-security/universe amd64 c-n-f Metadata [12.9 kB]
Get:30 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 DEP-11 Metadata [2,464 B]
Fetched 7,900 kB in 2s (3,326 kB/s)                                         
Reading package lists... Done
❯ sudo apt update
Hit:1 http://security.ubuntu.com/ubuntu focal-security InRelease
Hit:2 http://ca.archive.ubuntu.com/ubuntu focal InRelease                                                                      
Hit:3 http://ca.archive.ubuntu.com/ubuntu focal-updates InRelease                                                              
Hit:4 https://deb.nodesource.com/node_14.x focal InRelease                                                                     
Hit:5 http://ca.archive.ubuntu.com/ubuntu focal-backports InRelease            
Hit:6 https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb ./ InRelease
Reading package lists... Done
Building dependency tree       
Reading state information... Done
17 packages can be upgraded. Run 'apt list --upgradable' to see them.
❯ sudo apt install rancher-desktop
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  rancher-desktop
0 upgraded, 1 newly installed, 0 to remove and 17 not upgraded.
Need to get 293 MB of archives.
After this operation, 711 MB of additional disk space will be used.
Get:1 https://download.opensuse.org/repositories/isv:/Rancher:/dev/deb ./ rancher-desktop 0.7.0beta.1.52.1c15d27-75.1 [293 MB]
Fetched 293 MB in 57s (5,150 kB/s)                                                                                                 
Selecting previously unselected package rancher-desktop.
(Reading database ... 196737 files and directories currently installed.)
Preparing to unpack .../rancher-desktop_0.7.0beta.1.52.1c15d27-75.1_amd64.deb ...
Unpacking rancher-desktop (0.7.0beta.1.52.1c15d27-75.1) ...
Setting up rancher-desktop (0.7.0beta.1.52.1c15d27-75.1) ...
Processing triggers for mime-support (3.64ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.24-1ubuntu3) ...
```

**Comment by ericpromislow** [comment]
Install worked on a new OpenSUSE (Leap 15.3) machine.

**Review by ericpromislow — APPROVED** [review]
I needed newer systems -- both my ubuntu and suse systems hadn't been literally touched since March 2020 (21 months), so I reinstalled and installation worked fine.

File: docs/installing.md
Comment (ericpromislow) [inline docs/installing.md:132]: Getting a `sudo` prompt in the middle of `curl` output slightly misleading. Maybe `curl -s` would help, but I also prefer pre-warming `sudo`, like:
```
sudo echo ok
curl -s https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/Release.key | sudo apt-key add -
...
```

There might be more conventional ways of doing this.

**Comment by adamkpickering** [comment]
I've tested all three package formats - they all worked.

**Review by evertonlperes — APPROVED** [review]
LGTM 🚀 
Tested on Leap 15.3 and Ubuntu 20.04. Nicely done @adamkpickering

**Pull request merged**
